### DISCUSSION TITLE
Help! Change items order in an array does not affect the folders order

### DISCUSSION THREAD

**Discussion by aztack** [pmndrs/leva#407] [Q&A]
I want change the order of items in an array.  After swap two items, all array items is in expected order, but the folder in the panel is remain unchanged.

Here is the [Codesandbox](https://codesandbox.io/s/leva-render-array-forked-u559wl?file=/src/App.js)
1. Click the 'Up' button in the second item `[1]aa2`
2. `[1]aa2` is expected to move to the top. The log in the console shows the correct order
3. But the folders in the panel remain unchanged

I tried set the order option to the index in `userControls`, but nothing happened.

![image](https://user-images.githubusercontent.com/782871/199882984-fe842c98-7531-417b-ba0c-5383d6e60f7e.png)

Can some help me? I stuck on this for three days..

**Comment by geyang** [comment]
Did you get this resolved?

**Reply by geyang** [reply]
Crap, was just about to ask, then saw this comment from myself 🤦
